bedrock/mozorg/templates/mozorg/about/policy/transparency/includes/past-reports.html

```diff
@@ -5,16 +5,17 @@
 {# Add links to previous reports as new reports are published. #}
 
 <nav class="prev-reports-nav">
-  <h3>{{ _('Previous Reports') }}</h3>
+  <h3>Previous Reports</h3>
   <ul>
-    <li><a href="{{ url('mozorg.about.policy.transparency.jul-dec-2019') }}">{{ _('July-December 2019 Report') }}</a></li>
-    <li><a href="{{ url('mozorg.about.policy.transparency.jan-jun-2019') }}">{{ _('January-June 2019') }}</a></li>
-    <li><a href="{{ url('mozorg.about.policy.transparency.jul-dec-2018') }}">{{ _('July-December 2018') }}</a></li>
-    <li><a href="{{ url('mozorg.about.policy.transparency.jan-jun-2018') }}">{{ _('January-June 2018') }}</a></li>
-    <li><a href="{{ url('mozorg.about.policy.transparency.jul-dec-2017') }}">{{ _('July-December 2017') }}</a></li>
-    <li><a href="{{ url('mozorg.about.policy.transparency.jan-jun-2017') }}">{{ _('January-June 2017') }}</a></li>
-    <li><a href="{{ url('mozorg.about.policy.transparency.jul-dec-2016') }}">{{ _('July-December 2016') }}</a></li>
-    <li><a href="{{ url('mozorg.about.policy.transparency.jan-jun-2016') }}">{{ _('January-June 2016') }}</a></li>
-    <li><a href="{{ url('mozorg.about.policy.transparency.jan-dec-2015') }}">{{ _('January-December 2015') }}</a></li>
+    <li><a href="{{ url('mozorg.about.policy.transparency.jan-jun-2020') }}">January-June 2020 Report</a></li>
```

Comment:
Should "Report" be in the title here? It doesn't match the rest of the links. 

<img width="782" alt="image" src="https://user-images.githubusercontent.com/2692333/108098769-61c29e00-7049-11eb-89a9-2ee4c3bb5a2b.png">